### PR TITLE
feat(docs): Create Typography style page

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-39

* Removed elements from basics page
* New Style page: Typography

* [x] Dev LGTM
* [x] Dev LGTM
* [x] Design LGTM

Screenshot:

typography encoreui

### CODE REVIEW

File: demo/styles/typography.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+              For developers not using encore-ui-template,
+              <a href="https://github.com/rackerlabs/encore-ui/blob/master/src/rxApp/README.md">
+                  read the README.md file for rxApp
+              </a> on fonts for more information.
+          </li>
+      </ul>
+  </p>
+
+  <h3> Headings/Title Styles </h3>
+  <p>
```
Comment (CITguy) [inline demo/styles/typography.html]: I'm not sure the `<p>` is necessary.

File: demo/styles/typography.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                  </tr>
+              </tbody>
+          </table>
+      </div>
+
+      <rx-styleguide code-url="styles/typography/headings.html"></rx-styleguide>
+  </p>
+
+  <h3>Descriptions / Metadata</h3>
+  <p>
```
Comment (CITguy) [inline demo/styles/typography.html]: The `<p>` seems unnecessary.

File: demo/styles/typography.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+                  content - and you may want to consider a different design pattern. (See
+                  the "Non-Data based Tables" section in the Table page for more information.)
+              </li>
+          </ul>
+      </div>
+  </p>
+  <rx-styleguide code-url="styles/typography/metadata.html"></rx-styleguide>
+
+  <h3>When There is No Data to Display</h3>
+  <p>
```
Comment (CITguy) [inline demo/styles/typography.html]: The `<p>` seems unnecessary.

File: demo/styles/typography.html [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+          <li>
+              Note that a separate style exists to communicate
+              <a href="#/styles/wells"> when data isn't present due to permissions </a>,
+              or when an entire table is empty (see "Null Patterns" section of
+              <a href="#/styleguide/tables"> Tables style guide </a>).
+      </ul>
+  </p>
+
+  <h3>Lists</h3>
+  <p>
```
Comment (CITguy) [inline demo/styles/typography.html]: The `<p>` seems unnecessary.

File: demo/styles/typography/lists.html [hunk 5]
```diff
@@ -0,0 +1,37 @@
+<div layout="row">
+    <div flex>
```
Comment (CITguy) [inline demo/styles/typography/lists.html]: Nice use of the `layout` component! :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Comment by bry4n** [comment]
@lfielder or @glynnis or @ElementE can you review the design?

**Comment by glynnis** [comment]
LGTM

**Pull request merged**